pkg/cosign/verify.go

```diff
@@ -721,19 +803,19 @@ func verifyInternal(ctx context.Context, sig oci.Signature, h v1.Hash,
 			return false, err
 		}
 		// If there is no chain annotation present, we preserve the pools set in the CheckOpts.
+		var pool = (*x509.CertPool)(nil)
```

Comment:
```suggestion
		var pool *x509.CertPool
```

Reply:
This should be sufficient to create a pointer.

Reply:
Agreed i will incorporate this comment